code/mob/living/critter/flockcritter_parent.dm

```diff
@@ -136,8 +136,10 @@
 	src.flock.removeDrone(src)
 
 /mob/living/critter/flock/projCanHit(datum/projectile/P)
-	if(istype(P, /datum/projectile/energy_bolt/flockdrone))
+	if (istype(P, /datum/projectile/energy_bolt/flockdrone))
 		return FALSE
+	if (!isalive(src)) //we cant_lie but still want to have projectiles act as if we are lying when dead
```

Comment:
```suggestion
	if (isdead(src)) //we cant_lie but still want to have projectiles act as if we are lying when dead
```
Up to you, but idk might be safer with the other stat existing, and I think everywhere else in Flock code isdead() is used